lib/active_resource/railtie.rb

```diff
@@ -1,9 +1,13 @@
 require "active_resource"
 require "rails"
+require "active_resource/active_job_serializer"
 
 module ActiveResource
   class Railtie < Rails::Railtie
     config.active_resource = ActiveSupport::OrderedOptions.new
+    if defined? ActiveResource::ActiveJobSerializer
```

Comment:
Needs a line break just above and wrap argument in parens.

Reply:
I'm not sure if Active Job is loaded at this point. Consider a `boot.rb` that looks like:

```ruby
require "active_resource/railtie"
require "rails/all" # Now we get Active Job.
```

We should our framework load hooks

```ruby
initializer "active_resource.add_active_job_serializer" do
  ActiveSupport.on_load(:active_job) do
    if defined?(…)
      config.active_job.custom_serializers << …
    end
  end
end
```